Modules/Options/GeneralTab/QuestieOptionsGeneral.lua

```diff
@@ -287,6 +287,22 @@ function QuestieOptions.tabs.general:Initialize()
                             end
                         end,
                     },
+                    miniWorldMapCoordinatesEnabled = {
+                        type = "toggle",
+                        order = 5.55,
+                        disabled = function() return not Questie.db.profile.mapCoordinatesEnabled end,
+                        name = function() return l10n('Mini World Map Coord.') end,
+                        desc = function() return l10n("Place the Player's coordinates and Cursor's coordinates on the Mini World Map's title."); end,
```

Comment:
These two new strings need a corresponding translation entry in https://github.com/Questie/Questie/blob/master/Localization/Translations/Options/General.lua